matching-game/jest.setup.ts

```diff
@@ -0,0 +1 @@
+import '@testing-library/jest-dom'
```

Comment:
this just saves us the trouble of writing `import '@testing-library/jest-dom` in every damn test file. basically, we need this import so we have access to all of the test assertions.